crates/storage/libmdbx-rs/Cargo.toml

```diff
@@ -22,18 +22,22 @@ indexmap = "2"
 libc = "0.2"
 parking_lot.workspace = true
 thiserror.workspace = true
-dashmap = { version = "5.5.3", features = ["inline"], optional = true }
+dashmap = { version = "5.5.3", features = ["inline"] }
 tracing = { workspace = true, optional = true }
 
 ffi = { package = "reth-mdbx-sys", path = "./mdbx-sys" }
 
 [target.'cfg(not(windows))'.dependencies]
 libffi = "3.2.0"
 
+# metrics
+reth-metrics.workspace = true
+metrics.workspace = true
+
 [features]
 default = []
 return-borrowed = []
-read-tx-timeouts = ["dashmap", "dashmap/inline", "tracing"]
+read-tx-timeouts = ["tracing"]
```

Comment:
I believe `dashmap` dependency still should be feature-gated

Reply:
yes, this is remains of aborted rw set